README.md

```diff
@@ -17,6 +17,10 @@ This repo is made up of six (6) product sub directories:
 - [Netmanager](/netmanager/)
 - [Website](/website/)
 
+## .env.sample file
+
```

Comment:
Please move this change to this line of the netmanager README https://github.com/rohitkori/AirQo-frontend/blob/bd7cab7856e425c5722e650a363eea310a91dc78/netmanager/README.md?plain=1#L9